### PR TITLE
Implement ClientServiceBase.validateLedger

### PR DESCRIPTION
**Pull request by supl**
This PR implements validateLedger like https://github.com/scalar-labs/scalardl-javascript-sdk-base/blob/master/index.js#L362

### CODE REVIEW

File: node/common/builder/ledger_validation_request.ts [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-
-export type LedgerValidationRequest = {
-  setAssetId: (id: string) => void;
-  setStartAge: (startAge: number) => void;
-  setEndAge: (endAge: number) => void;
-  setCertHolderId: (certHolderId: string) => void;
-  setCertVersion: (certVersion: number) => void;
-  setSignature: (signature: Uint8Array) => void;
-};
+import {LedgerValidationRequest} from '../scalar.proto';
```
Comment (supl) [inline node/common/builder/ledger_validation_request.ts:3]: The type definitions are all in scalar.proto.ts

File: node/common/client_config.ts [hunk 2]
```diff
@@ -281,7 +281,7 @@ export class ClientConfig {
     ] as number;
   }
 
-  getAuditorLinearizableValidationContractId(): string | undefined | null {
+  getAuditorLinearizableValidationContractId(): string {
```
Comment (supl) [inline node/common/client_config.ts:284]: we will have a default value `validate-ledger` anyway, so this part can be simplified.

**Review by ypeckstadt — APPROVED** [review]
LGTM. Thank you.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by jnmt — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**